packages/core/src/browser/tree/tree.spec.ts

```diff
@@ -174,8 +174,7 @@ describe('Tree', () => {
       const expectedRefreshedNodes = new Set([
         retrieveNode<CompositeTreeNode>('1'),
         retrieveNode<CompositeTreeNode>('1.1'),
-        retrieveNode<CompositeTreeNode>('1.2'),
-        retrieveNode<CompositeTreeNode>('1.2.1')]);
```

Comment:
There is a race condition here where not all nodes are deleted before the `refresh()` promise returns.

Reply:
Had a hard time with this one but why this test was OK before migration to node 10 ?

Reply:
I think the race simply gets further in different Node versions :/